Author: Michel Dänzer <daenzer@debian.org>
Description: Link statically with libbfd.
--- a/configure
+++ b/configure
@@ -3893,7 +3893,7 @@
 { $as_echo "$as_me:$LINENO: result: $ac_cv_lib_bfd_bfd_get_error" >&5
 $as_echo "$ac_cv_lib_bfd_bfd_get_error" >&6; }
 if test "x$ac_cv_lib_bfd_bfd_get_error" = x""yes; then
-  DEP_LIBS="$DEP_LIBS -lbfd -liberty"
+  DEP_LIBS="$DEP_LIBS /usr/lib/libbfd.a -liberty"
 else
   { { $as_echo "$as_me:$LINENO: error: libbfd is required to compile sysprof" >&5
 $as_echo "$as_me: error: libbfd is required to compile sysprof" >&2;}
--- a/configure.ac
+++ b/configure.ac
@@ -90,7 +90,7 @@
   AC_CHECK_LIB(iberty, cplus_demangle_opname, [],
     AC_MSG_ERROR([libiberty is required to compile sysprof]), -ldl))
 
-AC_CHECK_LIB(bfd, bfd_get_error, [DEP_LIBS="$DEP_LIBS -lbfd -liberty"],
+AC_CHECK_LIB(bfd, bfd_get_error, [DEP_LIBS="$DEP_LIBS /usr/lib/libbfd.a -liberty"],
   AC_MSG_ERROR([libbfd is required to compile sysprof]),
   -liberty)
 
